### ISSUE TITLE
Command/Config Error

### ISSUE THREAD

**Issue by Mirrage0** [ralphkb/sentinel-tickets#6]
```plain
TypeError: Cannot read properties of undefined (reading 'support_role_ids')
    at checkSupportRole (C:\Users\Microwave\Desktop\bots\ticketbot\index.js:145:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.execute (C:\Users\Microwave\Desktop\bots\ticketbot\commands\Tickets\panel.js:32:28)
    at async Object.execute (C:\Users\Microwave\Desktop\bots\ticketbot\events\interactionCreate.js:128:9)
```
I get the above error in the console when using `/panel` command. I have correctly filled out the `config.yml` file too

Logging the `ticketCategories` object does show the `support_role_ids`.

Here is the function for reference:
```js
async function checkSupportRole(interaction) {
  const customIds = Object.keys(ticketCategories);
  let foundId;
  const ticketType = await ticketsDB.get(
    `${interaction.channel.id}.ticketType`,
  );
  for (const id of customIds) {
    if (ticketCategories[id].name === ticketType) {
      foundId = id;
      break;
    }
  }
  const allowedRoles = ticketCategories[foundId].support_role_ids;
  return interaction.member.roles.cache.some((role) =>
    allowedRoles.includes(role.id),
  );
}
```
`ticketType` object is undefined which is the lead cause of the error. It will always be undefined as `tickets.sqlite` file is empty.
Here is the below line for reference:
```js
await ticketsDB.get(
    `${interaction.channel.id}.ticketType`,
  )
```

**Comment by ralphkb** [comment]
Hello @Mirrage0, thank you for providing enough details to track down this issue.
This was indeed caused by the new checkSupportRole function which was meant to be for tickets commands only since it starts by checking the ticketType so it will not work for commands such as blacklist, unblacklist or the panel.

I've released a new version of the bot and changed this so that support role IDs for the panel command can now be defined in the config for it that got added instead. This should fix the issue and still allow for support role IDs to be defined for this specific command.

Thank you for using the bot and reporting issues! 😄
